spec/content.spec.js

```diff
@@ -169,6 +169,38 @@ describe('Content', () => {
     })
   })
 
+  describe('getTagsByNameAndAttributes()', () => {
+    let range
+    beforeEach(() => {
+      range = rangy.createRange()
```

Comment:
And here you can use `this.range = ...`. That's the default of how to set a variable with mocha.